sinks/api/types.go

```diff
@@ -129,6 +129,11 @@ type Point struct {
 	Value interface{}
 }
 
+type internalPoint struct {
+	labels map[string]string
```

Comment:
Can we document auto-population of these? I'm guessing we only set the specified labels and may set the values of the rest